src/app/@modules/main-list/main-list.component.html

```diff
@@ -1,6 +1,6 @@
 <app-search-box (searchOnChange)="searchOnChange($event)"></app-search-box>
 
-<section class="content-section" *ngIf="searchRequest$ | async as searchRequest">
+<section class="content-section-landing" *ngIf="searchRequest$ | async as searchRequest">
```

Comment:
Add Element from BEM `content-section content-section__landing`.
Relates to the comment I left in CSS file.